moveit_core/collision_detection_fcl/include/moveit/collision_detection_fcl/collision_env_fcl.h

```diff
@@ -95,14 +95,20 @@ class CollisionEnvFCL : public CollisionEnv
   void setWorld(const WorldPtr& world) override;
 
 protected:
-  /** \brief Updates the collision objects saved in the manager to reflect a new padding or scaling of the robot links
-   */
+  /** \brief Updates the FCL collision geometry and objects saved in the CollisionRobotFCL members to reflect a new
+  *   padding or scaling of the robot links.
+  *
+  *   It searches for the the link through the pointed to robot model of the CollisionRobot and then constructs new FCL
+  *   collision objects and geometries depending on the changed robot model.
+  *
+  *   \param links The names of the links which have been updated in the robot model */
   void updatedPaddingOrScaling(const std::vector<std::string>& links) override;
 
   /** \brief Bundles the different checkSelfCollision functions into a single function */
   void checkSelfCollisionHelper(const CollisionRequest& req, CollisionResult& res, const robot_state::RobotState& state,
                                 const AllowedCollisionMatrix* acm) const;
 
+  /** \brief Common helper used by the different checkRobotCollision functions. */
```

Comment:
For what?

Reply:
There exists two different `checkRobotCollisionFunction`s with different arguments for CCD and discrete. The helpers implement the common functionality.

Reply:
Helpful explanation that I think won't hurt to add here.

Reply:
I added more details